pkg/okctl/okctl.go

```diff
@@ -171,31 +171,18 @@ func (o *Okctl) StateHandlers(nodes *clientCore.StateNodes) *clientCore.StateHan
 	}
 }
 
-// InitializeToolChain with core services
-func (o *Okctl) InitializeToolChain() (*clientDirectAPI.ToolChain, error) {
+// InitializeDirectClients with core services
+func (o *Okctl) InitializeDirectClients() (*clientDirectAPI.ToolChain, error) {
 	return &clientDirectAPI.ToolChain{
-		AppPostgresIntegration: clientDirectAPI.NewApplicationPostgresIntegrationAPI(o.coreServices.Kube),
-		Certificate:            clientDirectAPI.NewCertificateAPI(o.coreServices.Certificate),
-		Cluster:                clientDirectAPI.NewClusterAPI(o.coreServices.Cluster),
-		Component:              clientDirectAPI.NewComponentAPI(o.coreServices.ComponentService),
-		ContainerRepo:          clientDirectAPI.NewContainerRepositoryAPI(o.coreServices.ContainerRepositoryService),
-		ExternalDNS:            clientDirectAPI.NewExternalDNSAPI(o.coreServices.Kube),
-		Helm:                   clientDirectAPI.NewHelmAPI(o.coreServices.Helm),
-		IdentityManager:        clientDirectAPI.NewIdentityManagerAPI(o.coreServices.IdentityManager),
-		ManagedPolicy:          clientDirectAPI.NewManagedPolicyAPI(o.coreServices.ManagedPolicy),
-		Manifest:               clientDirectAPI.NewManifestAPI(o.coreServices.Kube),
-		Parameter:              clientDirectAPI.NewParameterAPI(o.coreServices.Parameter),
-		SecuityGroup:           clientDirectAPI.NewSecurityGroupAPI(o.coreServices.SecurityGroupService),
-		ServiceAccount:         clientDirectAPI.NewServiceAccountAPI(o.coreServices.ServiceAccount),
-		Vpc:                    clientDirectAPI.NewVPCAPI(o.coreServices.Vpc),
-		Domain:                 clientDirectAPI.NewDomainAPI(o.coreServices.Domain),
+		Helm:   clientDirectAPI.NewHelmAPI(o.toolChain.Helm),
+		Domain: clientDirectAPI.NewDomainAPI(o.toolChain.Domain),
```

Comment:
Did you forget to move this to the `InitializeToolchain()` function?

Reply:
No, they stay here until they can be refactored away completely.